Snippets/Testing/Testing_6/UpgradeGuides/6to7/WhenHandling.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace Testing_6.UpgradeGuides._6to7
```

Comment:
remove namespace

Reply:
@weralabaj happy to do this but all of the other classes in this solution have namespaces that match the folder. I can submit a separate PR to remove them all?

Reply:
OK, let's ignore for now then and we'll clean up separately